scrapscript.py

```diff
@@ -37,10 +37,15 @@ class Token:
 
 
 @dataclass(eq=True)
-class NumLit(Token):
+class IntLit(Token):
```

Comment:
Can do this rename from `NumLit` to `IntLit` in another PR if we want

Reply:
I think in a separate commit, not necessarily separate PR

Reply:
Eh, I'll do a separate PR to keep things cleaner: #69

Reply:
Done. This PR diff should be easier to read through now